src/Spanner/composer.json

```diff
@@ -5,9 +5,9 @@
     "minimum-stability": "stable",
     "require": {
         "ext-grpc": "*",
-        "google/cloud-core": "^1.4",
-        "google/gax": "^0.10",
-        "google/proto-client-php": "^0.13"
+        "google/cloud-core": "^1.6",
```

Comment:
Done - although this seems like a potential problem in our release process, if we cannot do a simultaneous release of two components, one of which depends on the new release of the other. It may be that this is simply a problem of composer, where in order to install a local package it is necessary to modify the composer file. We can discuss and try to resolve this outside of this PR.